recipe/meta.yaml

```diff
@@ -1,71 +1,64 @@
 {% set name = "prophet" %}
-{% set version = "1.0.1" %}
+{% set version = "1.1.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 3e682e8ea6e1ee26a92cf289f207d539f30e44879126c128ff8f4e360eb25a8b
+  sha256: f8d6d45a4391fdf740c7b74f79a72313f95a5f6e3417eaf1c68892b47b920646
 
 build:
   number: 0
-  # As of 3/4/2022, we don't have the deps built out to support a py310 build.
-  skip: true  # [py==310]
-  script: "{{ PYTHON }} -m pip install . -vv"
+  # LunarCalendar, which is a noarch package, requires ephem, unavailable on s390x
+  # https://blog.quantinsti.com/installing-prophet-library-windows/
+  skip: true  # [py<39 or not win]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   binary_has_prefix_files:
-    - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.pkl
+    - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.bin
+  # To use our own cmdstan rather than download and build a different one:
+  script_env:
+    - PROPHET_REPACKAGE_CMDSTAN=False
```

Comment:
note that on conda-forge it's `false` (lower-case). There's an error message for win `UserWarning: The environment variable 'PROPHET_REPACKAGE_CMDSTAN' is being passed through with value '<hidden>'.` Not sure if this is relevant to the build failure you're experiencing.